wp-includes/widgets/class-wp-widget-media-video.php

```diff
@@ -70,6 +70,12 @@ public function get_instance_schema() {
 					'default' => '',
 					'sanitize_callback' => 'wp_kses_post',
 				),
+				'poster' => array(
```

Comment:
This attribute name confused me at first ( could be lack of 2nd cup of ☕️  ) - at first glance I thought it was related to the original user that posted the video... the description and usage in the code cleared it up for me though.